sdk/tests/types/api/core.rs

```diff
@@ -0,0 +1,103 @@
+
+    assert!(u8::unpack::<_, true>(&mut unpacker, &()).is_err());
+
+    res
+}
+
+#[test]
+fn responses() {
+    // GET /api/routes
+    json_response::<RoutesResponse>("get-routes-response-example.json").unwrap();
+    // GET /api/core/v3/info
+    json_response::<InfoResponse>("get-info-response-example.json").unwrap();
+    // GET /api/core/v3/accounts/{bech32Address}/congestion
+    json_response::<CongestionResponse>("get-congestion-estimate-response-example.json").unwrap();
+    // GET /api/core/v3/rewards/{outputId}
+    json_response::<ManaRewardsResponse>("get-mana-rewards-example.json").unwrap();
+    // GET /api/core/v3/validators
+    json_response::<ValidatorsResponse>("get-validators-example.json").unwrap();
+    // GET /api/core/v3/validators/{bech32Address}
+    json_response::<ValidatorResponse>("get-validator-example.json").unwrap();
+    // GET /api/core/v3/committee
+    json_response::<CommitteeResponse>("get-committee-example.json").unwrap();
+    // GET /api/core/v3/blocks/issuance
+    json_response::<IssuanceBlockHeaderResponse>("get-buildingBlock-response-example.json").unwrap();
+    // POST /api/core/v3/blocks
+    json_response::<SubmitBlockResponse>("post-blocks-response-example.json").unwrap();
+    // GET /api/core/v3/blocks/{blockId}
+    json_response::<BlockDto>("get-block-by-id-empty-response-example.json").unwrap();
+    json_response::<BlockDto>("tagged-data-block-example.json").unwrap();
+    json_response::<BlockDto>("transaction-block-example.json").unwrap();
+    json_response::<BlockDto>("get-block-by-id-validation-response-example.json").unwrap();
+    // GET /api/core/v3/blocks/{blockId}/metadata
+    json_response::<BlockMetadataResponse>("get-block-by-id-response-example-new-transaction.json").unwrap();
+    json_response::<BlockMetadataResponse>("get-block-by-id-response-example-new.json").unwrap();
+    json_response::<BlockMetadataResponse>("get-block-by-id-response-example-confirmed-transaction.json").unwrap();
+    json_response::<BlockMetadataResponse>("get-block-by-id-response-example-confirmed.json").unwrap();
+    json_response::<BlockMetadataResponse>("get-block-by-id-response-example-conflicting-transaction.json").unwrap();
+    // GET /api/core/v3/blocks/{blockId}/full
+    json_response::<BlockWithMetadataResponse>("get-full-block-by-id-tagged-data-response-example.json").unwrap();
+    // GET /api/core/v3/outputs/{outputId}
+    json_response::<OutputResponse>("get-outputs-by-id-response-example.json").unwrap();
+    // GET /api/core/v3/outputs/{outputId}/metadata
+    json_response::<OutputMetadata>("get-output-metadata-by-id-response-unspent-example.json").unwrap();
+    json_response::<OutputMetadata>("get-output-metadata-by-id-response-spent-example.json").unwrap();
+    // GET /api/core/v3/outputs/{outputId}/full
+    json_response::<OutputWithMetadata>("get-full-output-metadata-example.json").unwrap();
+    // GET /api/core/v3/transactions/{transactionId}/metadata
```

Comment:
What about `/api/core/v3/transactions/{transactionId}/included-block` and `/api/core/v3/transactions/{transactionId}/included-block/metadata`?

Reply:
I guess it's covered by `// GET /api/core/v3/blocks/{blockId}/metadata`

Reply:
They reuse the same example as block and block/metadata